# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/text/reftest.list
# Commit: 5589923a52c8
# Full Hash: 5589923a52c80ed09c8ce383afb0385cabc0ec18
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r 2d192d43277b -r 5589923a52c8 gfx/wr/wrench/reftests/text/reftest.list
--- a/gfx/wr/wrench/reftests/text/reftest.list	Tue Dec 15 00:01:19 2020 +0000
+++ b/gfx/wr/wrench/reftests/text/reftest.list	Tue Dec 15 00:43:28 2020 +0000
@@ -61,7 +61,7 @@
 == shadow-clip.yaml shadow-clip-ref.yaml
 == shadow-fast-clip.yaml shadow-fast-clip-ref.yaml
 skip_on(android,device) == shadow-partial-glyph.yaml shadow-partial-glyph-ref.yaml  # Fails on Pixel2
-fuzzy(2,120) platform(linux) == shadow-transforms.yaml shadow-transforms.png
+fuzzy(2,135) platform(linux) == shadow-transforms.yaml shadow-transforms.png
 fuzzy(1,200) platform(linux) == raster-space.yaml raster-space.png
 skip_on(android) skip_on(mac,>=10.14) != allow-subpixel.yaml allow-subpixel-ref.yaml  # Android: we don't enable sub-px aa on this platform.
 skip_on(android,device) == bg-color.yaml bg-color-ref.yaml  # Fails on Pixel2